### ISSUE TITLE
Consider subpackage for various checks

### ISSUE THREAD

**Issue by nogoegst** [heptiolabs/healthcheck#15]
At the moment if one doesn't want to have specific checks from this package but they have to depend on them (e.g. prometheus) and vendor them accordingly.
It would be nice to have separate subpackages with handlers and with checks to avoid this situation.

**Comment by mattmoyer** [comment]
I agree that splitting out the metrics handler to a subpackage would be a good idea. That could avoid bringing in the prometheus dependency if you're not using it. The other checks are pretty lightweight in terms of dependencies, but splitting them out wouldn't hurt either.

**Comment by nogoegst** [comment]
@mattmoyer the problem here is that such a change will break API. At the moment there is no versioning in place (that could make it possible to just bump major version).
